### PR TITLE
release raiwidgets and responsibleai 0.14.0

### PR DESCRIPTION
**Pull request by imatiach-msft**
## v0.14.0

- educational materials
  - add Cognitive Service fairness assessment notebooks
- new features
  - ## Error Analysis
    - add new accuracy and f1-score metrics in erroranalysis python backend and UI frontend
    - add importances to ErrorReport for error analysis
    - make ErrorReport, PredictionsAnalyzer and ModelAnalyzer public in erroranalysis package
    - display message bar after saving a cohort in error analysis dashboard
  - ## Counterfactual
    - implement save and load for counterfactual manager
    - make load method uniform across all RAI managers
    - perform schema validations on serialized counterfactual output
  - ## Interpret
    - upgrade interpret-community to 0.22.0
- bug fixes
  - ## Model Analysis
    - remove drag and drop instructions in the dashboard navigation
    - add theme support for model statistics and dataset explorer
    - remove setup_explainer after ModelAnalysis load call in tests
    - add theme support for feature importances, causal, counterfactual tabs
    - hide opened feature list when clicking on the heatmap tab in model analysis
    - add localization for delete cohort dialog
  - ## Interpret
    - set is_run class variable in explainer manager
  - ## Causal
    - add init file to causal schema directory
  - ## Error Analysis
    - fix color for metrics that are not error metrics to be green in legend and heatmap nan cells
    - update features list in static view to have apply button removed and labels disabled for tree view parameters
    - update features list text for static case
    - fix running what if on categorical numeric values in error analysis dashboard
    - add theme support for error analysis view
    - fix for importances in static view causing failures when unspecified
    - fix feature list alignment issue
- ## other
  - add documentation for the release process
  - add E2E test for model analysis, running from python jupyter notebook and validating in cypress
  - sort all python imports using isort
  - add aggregate feature importance tests for responsibleaitoolbox
  - add E2E individual feature importance tests for responsibleaitoolbox-classification-model-assessment notebook
  - fix image links in responsibleaitoolbox-classification-model-assessment.ipynb notebook to reference images in folder instead of using embedded images directly
  - rename causal tests

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -24,6 +24,51 @@ this file to understand what changed.
...
+  - ## Error Analysis
+    - fix color for metrics that are not error metrics to be green in legend and heatmap nan cells
+    - update features list in static view to have apply button removed and labels disabled for tree view parameters
+    - update features list text for static case
+    - fix running what if on categorical numeric values in error analysis dashboard
+    - add theme support for error analysis view
+    - fix for importances in static view causing failures when unspecified
+    - fix feature list alignment issue
+  - ## Counterfactual
+    - perform schema validations on serialized counterfactual output
```
Comment (gaugup) [inline CHANGES.md:62]: This probably is not a bug fix. Maybe this could go in previous section?

File: CHANGES.md [hunk 2]
```diff
@@ -24,6 +24,51 @@ this file to understand what changed.
...
+  - ## Model Analysis
+    - remove drag and drop instructions in the dashboard navigation
+    - add theme support for model statistics and dataset explorer
+    - remove setup_explainer after ModelAnalysis load call in tests
+    - add theme support for feature importances, causal, counterfactual tabs
+    - hide opened feature list when clicking on the heatmap tab in model analysis
+    - add localization for delete cohort dialog
+  - ## Interpret
+    - set is_run class variable in explainer manager
+    - upgrade interpret-community to 0.22.0
```
Comment (gaugup) [inline CHANGES.md:50]: Also not a bug fix.

**Review by gaugup — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (imatiach-msft) [reply CHANGES.md:62]: done!

File: CHANGES.md (hunk 2 above)
Comment (imatiach-msft) [reply CHANGES.md:50]: done!

**Review by gaugup — APPROVED** [review]

**Review by vinuthakaranth — APPROVED** [review]

**Pull request merged**